src/psyclone/tests/test_files/dynamo0p3/testkern_one_int_scalar_mod.f90

```diff
@@ -0,0 +1,45 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please update this copyright notice to the one used in other files.

Reply:
Oh, and if you rename a file, it's good if you do it with `git mv oldfile newfile` as then git knows about it.

Reply:
Which copyright notice? They all have different ones.
Regarding the second comment, I assume that it is just that, a comment? Do _this_ next time?

Reply:
Yes, that was for future reference :-)